remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RemotingCommand.java

```diff
@@ -174,6 +176,34 @@ public CommandCustomHeader decodeCommandCustomHeader(
         return CodecHelper.decodeCommandCustomHeader(this, classHeader);
     }
 
+    public void makeCustomHeaderToNet() {
```

Comment:
Please use this  method  in CodecHelper class instead

Reply:
Some redundant code can be simplified,and use method in CodecHelper class instead if CodecHelper Class has relate method.